keyboards/cherrybstudio/cb87/keymaps/vial/rules.mk

```diff
@@ -0,0 +1,11 @@
+LTO_ENABLE = yes
+VIA_ENABLE = yes
+VIAL_ENABLE = yes
+TAP_DANCE_ENABLE = no
+COMBO_ENABLE = no
+KEY_OVERRIDE_ENABLE = no
+MOUSEKEY_ENABLE = no
+EXTRAKEY_ENABLE = no
+SPACE_CADET_ENABLE = no
+GRAVE_ESC_ENABLE = no 
+#define DYNAMIC_KEYMAP_LAYER_COUNT 2
```

Comment:
this is supposed to go into config.h

Reply:
Thanks for the heads-up. Reuploading changes on the disabled RGB lights too. 

There was a typo on the #undef function, saved a little bit of memory size.